src/core_plugins/i18n/server/i18n/index.js

```diff
@@ -0,0 +1,35 @@
+import  i18n from './i18n';
+
+/*
+ Manages the locale translations for Kibana. Responsible for loading translated content per locale.
+
+API:
+
+Register translations:
+  registerTranslations(<absolute_path_to_translation_file>)
+  The translation file will be bundled into one translation file per locale and stored in the Kibana data directory. Returns a Promise object.
```

Comment:
This comment is out of date now based on our decision to store file locations rather than pre-bundle contents